src/components/content-tab-settings.tsx

```diff
@@ -30,6 +32,7 @@ export function ContentTabSettings( { selectedSite }: ContentTabSettingsProps )
 	// Empty strings account for legacy sites lacking a stored password.
 	const storedPassword = decodePassword( selectedSite.adminPassword ?? '' );
 	const password = storedPassword === '' ? 'password' : storedPassword;
+	const phpVersion = useGetPhpVersion( selectedSite );
```

Comment:
I think the hook `useGetPhpVersion` is not necessary because the PHP version is already coming from the site details in `selectedSite`.
```
	const phpVersion = selectedSite.phpVersion ?? DEFAULT_PHP_VERSION;
```

Not sure if this approach has been followed based on the other hook `useGetWpVersion`. If so, [the WordPress version needs to be retrieved from the WordPress files as it's not referenced in the site details](https://github.com/Automattic/studio/blob/9da80ea926f6df3de22f3dc2497b5f7de66c03e4/src/ipc-handlers.ts#L468-L485). That's the main reason for needing that hook.

Reply:
I'll add these changes in a separate PR to avoid blocking this one.

Reply:
> I'll add these changes in a separate PR to avoid blocking this one.

https://github.com/Automattic/studio/pull/239